### PR TITLE
allow override of returned conference configuration

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

File: applications/ecallmgr/src/ecallmgr_fs_config.erl [hunk 1]
```diff
@@ -379,16 +379,35 @@ get_node_gateways(Node) ->
...
+-spec fix_conference_profile(kz_json:key(), kz_json:object()) -> {kz_json:key(), kz_json:object()}.
+fix_conference_profile(Name, Profile) ->
+    Routines = [fun maybe_fix_profile_tts/1
+               ,fun maybe_set_verbose_events/1
+               ],
+    {Name, kz_json:exec(Routines, Profile)}.
+
+-spec maybe_set_verbose_events(kz_json:object()) -> kz_json:object().
+maybe_set_verbose_events(Profile) ->
+    case kz_util:is_true(ecallmgr_config:get_boolean(<<"force_conference_verbose_events">>)) of
```
Comment (jamesaimonetti) [inline applications/ecallmgr/src/ecallmgr_fs_config.erl]: `is_true` after `get_boolean` seems redundant, no?

Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_config.erl]: lol, absolutely

Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_config.erl]: humm, actually its not
```
get_boolean(Key) ->
    get_boolean(Key, 'undefined').

get_boolean(Key, Default) ->
    case get(Key, Default) of
        Default -> Default;
```

Comment (lazedo) [inline applications/ecallmgr/src/ecallmgr_fs_config.erl]: but will amend to `is_true`

**Comment by jamesaimonetti** [comment]
Merged

**Pull request closed without merging**